### ISSUE TITLE
"Tell me a joke" intent

### ISSUE THREAD

**Issue by awallin** [mozilla/firefox-voice#511]
When someone asks for a joke:

- Tell me a joke
- Say something funny
- Make me laugh

Load the following.
https://www.youtube.com/watch?v=N3jx4WIUYy4

**Comment by ianb** [comment]
It would be fine to open a tab with the video. This could go in the `self.js` set of intents.

**Comment by maitrella** [comment]
I'm really new to the project and it's a very early version of the intent. Any comment will be welcome. :)

```js
  this.intentRunner.registerIntent({
    name: "self.tellJoke",
    description:
      "Someone ask for a joke or something funny",
    examples: ["Tell me a joke", "Say something funny", "Make me laugh"],
    match: `
    tell (me |) a joke
    say something funny
    make me laugh
    `,
    async run(context) {
      await browser.tabs.create({
        url: browser.runtime.getURL("https://www.youtube.com/watch?v=N3jx4WIUYy4"),
      });
    },
  });
```

**Comment by ianb** [comment]
@maitrella That looks good, ready to be a PR!

**Comment by maitrella** [comment]
PR #599 done

**Comment by awallin** [comment]
Implemented in #599
